CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
   The only real difference between those two queries is that the `Holding` query accepts either a denom or marker address.
 * Update the third-party protos and swagger files after the cosmos v0.46 bump. [#1017](https://github.com/provenance-io/provenance/issues/1017)
 * Stop using the deprecated Wrap and Wrapf functions in the sdk/types/errors package in favor of those functions off specific errors, or else the cosmossdk.io/errors package. [#1013](https://github.com/provenance-io/provenance/issues/995)
+* Add proposal to update msgfees denom metadata
```

Comment:
I feel like this should be in the `SDK` instead of our stuff. Technically, it's not specific to `x/msgfees`, but rather to the `x/bank` module.

I was actually surprised to see that they didn't have anything for doing this. I know the `x/marker` module provides ways to update their denom metadata entries, but there's no generic way outside of the genesis file.